iphone/Classes/TiUIOptionDialogProxy.m

```diff
@@ -99,17 +101,22 @@ -(void)show:(id)args
             }
             curIndex++;
         }
-        
+
         if ([TiUtils isIPad] && (cancelButtonIndex == -1)) {
             UIAlertAction* theAction = [UIAlertAction actionWithTitle:@"Cancel"
                                                                 style:UIAlertActionStyleCancel
                                                               handler:^(UIAlertAction * action){
                                                                   [self fireClickEventWithAction:action];
                                                               }];
+            if (!IS_NULL_OR_NIL(imagesOptions)) {
+                UIImage *image = [ [UIImage imageNamed:[imagesOptions objectAtIndex:curIndex]] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
+                [theAction setValue:image forKey:@"image"];
+            }
+
```

Comment:
This code would only be executed, if the current device is an iPad and no ``cancel`` property is set. It also does not check for array bounds (use case: options = ["test1","test2"] and imageOptions = ["image.png"]). So:
- Rename ``imagesOptions`` to ``imageOptions``
- Check the array bounds
- Use the TiUtils helper to create an image. It makes it save for the new asset catalog coming by default in 5.2.0.

Example:
```
NSMutableArray *imagesOptions = [self valueForKey:@"imageOptions"];

...
 
if (imagesOptions != nil && curIndex < [imagesOptions count]) {
  UIImage *image = [ [TiUtils image:[imagesOptions objectAtIndex:curIndex] proxy:self] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
  [theAction setValue:image forKey:@"image"];
}
```